doto-backend/src/routes/task-route.js

```diff
@@ -1,52 +1,16 @@
 const express = require('express');
 const router = express.Router();
-const User = require('../models/User');
 const Task = require('../models/Task');
 
-
-// GET ALL users
-
-router.get('/users', function(req, res){
-    let users = User.find({}, function(err, users){
-        if(err){
-            console.log(err);
-            res.status(400).json({msg: "failed"})
-        }
-        else {
-            console.log(users)
-            res.status(200).json(users);
-        }
-    })
-})
-
-// ADD user
-router.post('/users/add', function (req, res) {
-
-    let user = new User();
-    user.name = req.body.name;
-    user.picture = req.body.picture;
-    user.themePreference = req.body.themePreference;
-
-    user.save(function(err){
-        if(err){
-            console.log(err);
-            res.status(400).json({msg: "failed..."})
-        }
-        else{   
-            res.status(200).json(user)
-        }
-    });
-});
-
 //GET ALL task
-router.route('/schedule/get/:user').get((req,res) => {
+router.route('/get/:user').get((req,res) => {
```

Comment:
Is there a reason why this is `router.route(...).get(...)` instead of `router.get(...)` like the other routes?

Reply:
@jonathanvenema98 i think you wrote this, was there a reason it was router.route?

Reply:
@jonathanvenema98 said that it doesn't matter because there is still a (get), we can refactor this at the end :)